audio_service/pubspec.yaml

```diff
@@ -9,22 +9,22 @@ environment:
 
 dependencies:
   # Use these deps for local development
-  # audio_service_platform_interface:
-  #   path: ../audio_service_platform_interface
-  # audio_service_web:
-  #   path: ../audio_service_web
-
-  # Use these deps when pushing to origin (for the benefit of testers)
   audio_service_platform_interface:
-    git:
-      url: https://github.com/ryanheise/audio_service.git
-      ref: one-isolate
-      path: audio_service_platform_interface
+    path: ../audio_service_platform_interface
```

Comment:
This is indeed the intention but make sure you don't actually push this dep to origin. Leave this change out when you're ready to push.

Reply:
it's WIP so i didn't worry too much about that